ingress/controllers/gce/README.md

```diff
@@ -409,95 +395,121 @@ if ($http_x_forwarded_proto = "http") {
 }
 ```
 
-And you can try with the [https_example](https_example/README.md):
-```console
-$ cd https_example
-$ make keys secret
-# The CName used here is specific to the service specified in nginx-app.yaml.
-openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/tls.key -out /tmp/tls.crt -subj "/CN=example.com/O=example.com"
-Generating a 2048 bit RSA private key
-...........+++
-....................................................+++
-writing new private key to '/tmp/tls.key'
------
-godep go run make_secret.go -crt /tmp/tls.crt -key /tmp/tls.key > /tmp/tls.json
-```
-
-This will generate a secret in `/tmp/tls.json`, first create it
+Here's an example that demonstrates it, first lets create a self signed certificate:
 
 ```console
-$ kubectl create -f /tmp/tls.json
-$ kubectl describe secret tls-secret
-Name:		tls-secret
-Namespace:	default
-Labels:		<none>
-Annotations:	<none>
+$ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/tls.key -out /tmp/tls.crt -subj "/CN=foobar.com"
```

Comment:
Add a warning about the 365 day expiration cliff - some folks will convert this example to production use without realizing the expiration cliff exists.